src/helm/benchmark/presentation/summarize.py

```diff
@@ -1063,21 +1100,49 @@ def _adapter_spec_sort_key(spec):
 
         table = Table(title=title, header=header, rows=rows, links=links, name=name)
 
-        if add_win_rate:
-            # add overall win rate as the second column
-            WIN_RATE_AGGREGATION = "mean"
-            win_rates = compute_aggregate_row_win_rates(table, aggregation=WIN_RATE_AGGREGATION)
-            description = "How many models this model outperform on average (over columns)."
-            table.header.insert(
-                AGGREGATE_WIN_RATE_COLUMN,
-                HeaderCell(
-                    f"{WIN_RATE_AGGREGATION.capitalize()} win rate",
-                    description=description,
-                    lower_is_better=False,
-                ),
-            )
-            for row, win_rate in zip(table.rows, win_rates):
-                row.insert(AGGREGATE_WIN_RATE_COLUMN, Cell(win_rate))
+        selected_agg_strats = selected_agg_strats if selected_agg_strats is not None else ["win_rate"]
+
+        # this preserves backwards compatibility for self.schema.name_to_metric_group[metric_group].hide_win_rates
+        # hide_win_rate is the inverse of add_win_rate here (see the function call for create_group_table)
+        hide_aggregation = not add_win_rate
+        if hide_aggregation:
+            selected_agg_strats = []
+        insertion_column = AGGREGATE_WIN_RATE_COLUMN
```

Comment:
This insertion introduces a bug where using multiple aggregation strategies causes every aggregate after the first to be incorrect, because they will consider the previously computed aggregates as input values.

The better way to do this is to do

```python
aggregate_header_cells: List[HeaderCell] = []
aggregate_rows: List[List[HeaderCell]] = []
```

and then concatenate these cells to the original cells at the end. Then you should be able to delete `AGGREGATE_WIN_RATE_COLUMN` as well.

Or you could just limit users to a single aggregate column to avoid this complexity.

Reply:
I opted to do a version of your suggestion as opposed to limiting users. Let me know if you have a preference for limiting users though.